lib/storage/src/chunks/getChunkBuffer.ts

```diff
@@ -0,0 +1,22 @@
+import { RawDataPart } from "../Upload";
+
+export async function* getChunk(data: Buffer, partSize: number): AsyncGenerator<RawDataPart> {
+  let partNumber = 1;
+  let startByte = 0;
+  let endByte = startByte + partSize;
```

Comment:
The value in `startByte` is 0 and is not required to be added to `endByte`.
```suggestion
  let endByte = partSize;
```

Reply:
done